File: JsCore/JavaScript Advanced/Classes and Class Members/Person.js
class Person {
    constructor (firstName, lastName, age, email) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.age = age;
        this.email = email;
    }
    toString() {
        return `${this.firstName} ${this.lastName} (age: ${this.age}, email: ${this.email})`;
    }
}
let person = new Person('Vladimir', 'Gadjov', 26, 'vladko_gadjov@abv.bg');
console.log(person.toString());

File: C# Fundamentals/C# OOP Basics/Encapsulation Lab/1. Sort Persons by Name and Age/Person.cs
public class Person
{
    public Person(string firstName, string lastName, int age)
    {
        this.FirstName = firstName;
        this.LastName = lastName;
        this.Age = age;
    }

    public string FirstName { get; set; }

    private string LastName { get; set; }

    public int Age { get; set; }

    public override string ToString()
    {
        return $"{this.FirstName} {this.LastName} is a {this.Age} years old";
    }
}
